server/config/memories/memoryModel.js

```diff
@@ -0,0 +1,12 @@
+var mongoose = require('mongoose');
+var Schema = mongoose.Schema;
+
+var memorySchema = new Schema({
+  title: String,
+  filePath: String,
+  createdAt: Date,
+  /* TODO: How will we store this data? JSON? stringified? tags? */
+  analyses: []
```

Comment:
Hard to say before using the APIs :) but I imagine a decision we'll need to make is either one schema that's broad enough to represent all analyses or a schema per analysis?